poncho/src/poncho_package_serverize

```diff
@@ -7,13 +7,14 @@ if __name__ == "__main__":
     parser.add_argument('--function', action='append', help='Name of the function in filename to be serverized')
     parser.add_argument('--pack', help='Optional argument. If given, will create an environment to run the network function at this path')
     parser.add_argument('--dest', help='Destination path for the output network function')
+    parser.add_argument('--version', choices=['work_queue', 'task_vine'], help='Version of serverless (work_queue or task_vine)', required=True)
```

Comment:
A very little thing: we use `taskvine` consistently instead of `task_vine`

Reply:
And related, use `vine` instead of `tv`.